Doc/library/pathlib.rst

```diff
@@ -755,6 +755,30 @@ call fails (for example because the path doesn't exist).
    .. versionchanged:: 3.10
       The *follow_symlinks* parameter was added.
 
+.. method:: Path.chown(uid, gid, *, dir_fd=None, follow_symlinks=True)
+
+   Change the file ownership, like :func:`os.chown`.
+
+   This method normally follows symlinks. Some Unix flavours support changing
+   permissions on the symlink itself; on these platforms you may add the
+   argument ``follow_symlinks=False``, or use :meth:`~Path.lchown`.
```

Comment:
Why do we need two ways to do the same thing?

Reply:
I do not think, that we essentially need both functions. But since both are available in `os` and both `chmod` and `lchmod` are available in `pathlib` I adapted this behavior for `chown`.